src/Chapter2.hs

```diff
@@ -401,7 +401,7 @@ always matches (the same as a variable), but we don't use its value.
   the breakpoint, 'case' in Haskell always returns only a single
   expression for a single branch. You can think of this process as
   trying to match all patterns from the first one to the last one and
-  returning the expression on the right side of "=" only for the pattern
+  returning the expression on the right side of "->" only for the pattern
```

Comment:
Now when I look closer I see that we are not talking about `case-of` in here, but about _casing_ in Haskell as in the example above.
Maybe putting that instead of the case above would make it clearer.
We are talking about `cafe-of` with arrows in the following paragraph 🙂